tests/src/test/java/alluxio/testutils/LocalAlluxioClusterResource.java

```diff
@@ -219,11 +229,18 @@ public Builder setProperty(PropertyKey key, Object value) {
       return this;
     }
 
+    public Builder setSockets(ServerSocket rpcBindSocket, ServerSocket webBindSocket) {
```

Comment:
split into two setters?

Reply:
The way this is currently implemented is that both sockets need to be non-null in order for the master creation to respect the sockets. I put both in a setter to avoid the case where someone forgets to set the 2nd socket